### DISCUSSION TITLE
[Q] static plugins supports basic url

### DISCUSSION THREAD

**Discussion by zaishui15** [roadrunner-server/roadrunner#836] [Q&A] [answered]
```
static:
    basic_url: basic
    dir: src/public
```
I want to add “basic_url” config to static plugins

When  I visit  http://127.0.0.1/basic/file.log

In fact, the file path I visited was "src / public / file.log", not "src / public / basic / file.log"

**Comment by rustatian** [comment, marked as answer]
@zaishui15 Hey, you can use a `fileserver` plugin. It serves directories with its own http prefixes: https://github.com/spiral/roadrunner-plugins/blob/master/fileserver/docs/fileserver.md

**Reply by victor-sudakov** [reply]
Is the new fileserver plugin going to support the "allow" and "forbid" directives? They were kind of useful. https://github.com/spiral/roadrunner-plugins/issues/161

**Reply by zaishui15** [reply]
fileserver plugin can be implemented, but fileserver plugin must be a new HTTP service

**Reply by rustatian** [reply]
Fileserver plugin already exists as a separate plugin with a separate http server. https://github.com/spiral/roadrunner-plugins/blob/master/fileserver/docs/fileserver.md

**Reply by zaishui15** [reply]
I didn't want to separate http server

**Reply by rustatian** [reply]
Unfortunately, the `static` plugin will not receive this feature, because it's bound to the root http prefix.

**Reply by zaishui15** [reply]
thanks，I'll try to solve it with fileserver plugin
